<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>book_borrow</class>
 <widget class="QDialog" name="book_borrow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>图书编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>图书名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="book_ISBN">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="book_name_edit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="book_status">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>预借</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>借书</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>续借</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>还书</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="borrow">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="imformation">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
